code/mob/living/carbon/human.dm

```diff
@@ -83,7 +83,10 @@
 	var/yeet_chance = 0.1 //yeet
 
 	var/decomp_stage = DECOMP_STAGE_NO_ROT
+	var/no_miasma = FALSE
+	var/no_decomp = FALSE
 	var/time_until_decomposition = 0
+	var/silent_death = FALSE // Stops deathgasping and clown confetti
```

Comment:
I feel like these being simple mob properties would be better as they are gonna be FALSE 99% of the time.
See for example the `PROP_MOB_SUPPRESS_LAYDOWN_SOUND` for something similar I did not too long ago.